python/cuspatial/cuspatial/_lib/spatial.pxd

```diff
@@ -9,33 +9,44 @@ from cudf._lib.cudf cimport *
 from libcpp.pair cimport pair
 
 cdef extern from "point_in_polygon.hpp" namespace "cuspatial" nogil:
- cdef gdf_column point_in_polygon_bitmap(const gdf_column& pnt_x,
-                                         const gdf_column& pnt_y,
-                                         const gdf_column& ply_fpos,
-                                         const gdf_column& ply_rpos,
-                                         const gdf_column& ply_x,
-                                         const gdf_column& ply_y) except +
+    cdef gdf_column point_in_polygon_bitmap(
+        const gdf_column& pnt_x,
+        const gdf_column& pnt_y,
+        const gdf_column& ply_fpos,
+        const gdf_column& ply_rpos,
+        const gdf_column& ply_x,
+        const gdf_column& ply_y
+    ) except +
 
-cdef extern from "coordinate_transform.hpp" namespace "cuspatial" nogil: 
- cdef pair[gdf_column, gdf_column] lonlat_to_coord(const gdf_scalar cam_x,
-                                                   const gdf_scalar cam_y,
-                                                   const gdf_column  & in_x,
-                                                   const gdf_column  & in_y) except +
+cdef extern from "coordinate_transform.hpp" namespace "cuspatial" nogil:
+    cdef pair[gdf_column, gdf_column] lonlat_to_coord(
+        const gdf_scalar& cam_x,
+        const gdf_scalar& cam_y,
```

Comment:
These weren't pass by reference before, but the `.hpp` file had them as pass by reference so I updated it. If incorrect please let me know.